src/main/kotlin/net/axay/kotlinkitapi/implementation/Archer.kt

```diff
@@ -0,0 +1,53 @@
+package net.axay.kotlinkitapi.implementation
+
+import net.axay.kotlinkitapi.api.Kit
+import net.axay.kotlinkitapi.api.KitProperties
+import org.bukkit.Material
+import org.bukkit.entity.AbstractArrow
+import org.bukkit.entity.Arrow
+import org.bukkit.entity.LivingEntity
+import org.bukkit.entity.Player
+import org.bukkit.event.entity.EntityShootBowEvent
+import org.bukkit.event.entity.ProjectileHitEvent
+import org.bukkit.inventory.ItemStack
+import org.bukkit.potion.PotionEffect
+import org.bukkit.potion.PotionEffectType
+import java.util.*
+import kotlin.math.roundToInt
+
+class ArcherProperties : KitProperties() {
+    val effectDuration by int(80)
+    val effectMultiplier by int(0)
+    val arrowAmount by int(6)
+    val likelihood by int(50)
+    val potionEffectTypes = listOf(
+            PotionEffectType.BLINDNESS,
+            PotionEffectType.SLOW_FALLING,
+            PotionEffectType.LEVITATION,
+            PotionEffectType.REGENERATION,
+            PotionEffectType.SLOW_DIGGING,
+            PotionEffectType.HUNGER,
+            PotionEffectType.CONFUSION,
+            PotionEffectType.GLOWING,
+            PotionEffectType.WITHER,
+            PotionEffectType.POISON,
+            PotionEffectType.INCREASE_DAMAGE)
```

Comment:
fix formatting

Reply:
wie

Reply:
in eine reihe?

Reply:
nein die klammer am ende ist falsch drück einfach strg + alt + L zum formatieren